VagrantProvisionCentOS7.sh

```diff
@@ -286,12 +286,16 @@ fi
 
 # Check for a hoot Db
 if ! sudo -u postgres psql -lqt | grep -iw --quiet $DB_NAME; then
-    echo "### Creating Services Database..."
+    echo "### Creating Main Services Database..."
```

Comment:
This is a bit of a cleanup. I was getting Vagrant failures when it tried to re create the wfsstoredb

Reply:
Didn't we remove WFS Store or something?